### PR TITLE
Initial system integration code

### PR DESCRIPTION
**Pull request by MathewAaron**
This PR has the following code : 

- GPS sensor driver
- Camera Integration
- ROS wrappers for : GPS triggers, Biomass estimation and Biomass Map
- Unit and functional tests for all modules.

### CODE REVIEW

File: src/biomass_estimator_driver/scripts/biomass_estimator.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import json
+import os
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
+
+def biomassCorrelation(pix_count: float) -> float:
```
Comment (mikahpinegar) [inline src/biomass_estimator_driver/scripts/biomass_estimator.py]: should this be `snake_case`?

Comment (MathewAaron) [reply src/biomass_estimator_driver/scripts/biomass_estimator.py]: Yep, this is part of Daniel's code, I will edit it to a standard code template and get back.

Comment (MathewAaron) [reply src/biomass_estimator_driver/scripts/biomass_estimator.py]: Resolved #[6700027](https://github.com/precision-sustainable-ag/PlantMap3D-ROS/pull/38/commits/670002793c1804e2336a64a1605eebddef8bcfca)

**Comment by MathewAaron** [comment]
@mikahpinegar : Resolved merge conflict

**Pull request closed without merging**